index.js

```diff
@@ -334,7 +334,7 @@ const inlineCss = async opt => {
         style = document.createElement("style");
       style.type = "text/css";
       style.appendChild(document.createTextNode(allCss));
-      head.appendChild(style);
+      head && head.appendChild(style);
```

Comment:
this will hide potential error. I would prefer visible error instead of hiding

Reply:
Maybe a console.warning instead? or we could just append it to the end of the document rather than the head if the head isnt there?

Reply:
if the head isn't there it will not work as expected, where shall we insert those instead? We can't just ignore it, the whole thing could fail. What we can do is to create more user friendly error, like "Hey, we use head to do this manipulation, but we failed to find one, so tool doesn't work. Please add head"

Reply:
May I ask why there is no head in the first place?

Reply:
So this is not typical SPA application, like React or Vue. And I still don't understand what it does. I can not optimise for the case which I don't understand. There is no test for this case, anybody can change this in the future and even don't notice it. 

You know you can install fork from Github directly, right?

Reply:
@stereobooster Im not asking to optimize for this specific case, I'm asking to fix a bug where a null reference exception occurs.

Reply:
yes, but this fix also can hide error for other case. Fix would be to show human readable error

Reply:
@stereobooster Sure, but also just the fact that adding a style element when there are no linked stylesheets is not necessary, is it?

Reply:
> Sure, but also just the fact that adding a style element when there are no linked stylesheets is not necessary, is it?

For sure, but you can check exactly this case `if (criticalCss.length === 0) { return; }`.

Reply:
I will update my PR in the next few days